CHANGELOG.md

```diff
@@ -6,6 +6,20 @@
 
 All notable changes to this project will be documented in this file.
 
+## 4.2.0
+
+### Changes
+* feat: migrate owncloud guests-users to nextcloud guests by @skjnldsv in https://github.com/nextcloud/guests/pull/1286
+* feat(deps): Add Nextcloud 32 support by @nickvergessen in https://github.com/nextcloud/guests/pull/1282
```

Comment:
Hmm not sure you want to release this already?

Reply:
We usually do no ?
if in alpha/beta, it won't matter anyway

Reply:
When we'll get closer to 32, we'll have to release a new one anyway which will double check if still compatible

Reply:
What else are we supposed to do, shall we revert https://github.com/nextcloud/guests/pull/1282 ?

Reply:
yeah in some apps I stopped merging them :see_no_evil: 
For multi branch it's really annoying, either during development or releasing.

Reply:
Maybe a fix would be in server to allow enabling without `--force` when you run from git?

Reply:
I mean, isn't force what we expect to have for our daily anyway?
That's exactly the reason we have that option, no?

I actually don't get Tobias's comment on #1282  :thinking: